src/components/DayPicker.jsx

```diff
@@ -71,8 +75,10 @@ const defaultProps = {
 export default class DayPicker extends React.Component {
   constructor(props) {
     super(props);
+
+    this.hasSetInitialVisibleMonth = !!props.focused;
     this.state = {
-      currentMonth: moment(),
+      currentMonth: props.focused ? props.initialVisibleMonth() : moment(),
```

Comment:
sure, that works fine! just so it's harder to screw up the logic later, when it's in two places